specification/communications-manager.yaml

```diff
@@ -19,7 +19,8 @@ info:
     The Communications Manager API is currently not in an integration or production environment.
 
     ## Technology 
-    TBC
+    This API is a [REST-based](https://digital.nhs.uk/developer/guides-and-documentation/our-api-technologies#basic-rest) API.
```

Comment:
Enter an extra blank line in between these values so we get them broken up when this renders, see:

![image](https://github.com/NHSDigital/communications-manager/assets/879532/b07a5ff0-b286-42b1-adff-6551132df8d2)

Reply:
@PhilipSkinner I've (hopefully!) added another line using > rather than \n as discussed.

Reply:
Amended as requested now - thanks for the info.

Reply:
Thanks, this looks like this now:

![image](https://github.com/NHSDigital/communications-manager/assets/879532/4dafdece-a38c-4905-858d-531d1a172a16)